### ISSUE TITLE
Unable to get data anymore

### ISSUE THREAD

**Issue by kickouille** [nechry/elco-remocon-net-appdaemon#18]
Hello,

Last data received for me was yesterday evening at at around 19h30 CEST.
I tried connecting to https://www.remocon-net.remotethermo.com/, I'm able to login, but here also details are not loading.

Im wondering if I've been blacklisted or if it is a general problem.

Logs from appdaemon:
```
appdaemon    | 2024-09-24 14:37:58.955544 INFO remocon: Fetching remocon data...
appdaemon    | 2024-09-24 14:38:24.737281 INFO remocon: Unhandled exception: 'data'
appdaemon    | 2024-09-24 14:38:24.749525 WARNING AppDaemon: Excessive time spent in callback 'get_remocon_data() in remocon', Thread 'thread.thread-0' - now complete after 26.748527 seconds (limit=10)
```

Regards,

Kickouille

**Comment by nechry** [comment]
hello, from my side I just take a look and everithing work as expected. I also updated to the latest addon appdaemon release of yesterday.
If in remocon-net web portal you can't get details, for sure the AppDaemon will not as it use same api.
check also with the remocon-net mobile app.

**Comment by kickouille** [comment]
Hi Jean-François,

This is fixed.
I indeed did a test once with the App and it seemed to work, reason why I asked if there was a general issue, because that didn't seem consistent.
The gateway was connected to the WiFi, and I didn't suspect it could be the issue.
After power cycling the whole Elco infrastructure, it took a while to come back, but it finally worked.
So there was probably something stuck in there. I also noticed that WiFi signal for the gateway was weak on the AP, and after restarting it came back strong. I didn't check on which AP it was associated, but it probably did on a far away one, and restarting forced using one closer. It could also be the issue.

Regards,

Yanik
